Revert-tvOS/Sources/CollectionDataSource.swift

```diff
@@ -31,9 +32,28 @@ final class CollectionDataSource<Object: Collectable, Cell: UICollectionViewCell
     self.configureCell(cell, item: item)
     return cell
   }
+
+  subscript(indexPath: NSIndexPath) -> HomeItem {
+    get {
+      return self.collection[indexPath]
+    }
+  }
+
+  func clearFilter() {
+    self.collection = self.completeCollection
+  }
+
+  func filter(filterClosure: ((HomeItem) -> Bool)) {
```

Comment:
Can get rid of the exterior parenthesis I reckon, as long as it doesn't need to be optional - in which case having a `typealias` would probably be the way to go.